source/utils/paragraphHelper.py

```diff
@@ -73,8 +79,7 @@ def _isLastLineOfParagraph(line: str) -> bool:
 
 def _splitParagraphIntoChunks(paragraph: str) -> List[str]:
 	CHUNK_SIZE = 2048
-	SENTENCE_TERMINATOR = ". "
-	TERMINATOR_LEN = len(SENTENCE_TERMINATOR)
+	SENTENCE_TERMINATORS: FrozenSet[str] = {".", "?", "!"}
```

Comment:
This is how to actually create a frozenset, as opposed to hinting towards the type

```suggestion
	SENTENCE_TERMINATORS = frozenset({".", "?", "!"})
```